lib/core/ui/dashboard/home/home_tabs.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:live_score/core/extensions/localization/app_localizations_context.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+import 'package:live_score/features/dashboard/cubit/dashboard_cubit.dart';
+
+class HomeTabs extends StatelessWidget {
+  const HomeTabs({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final cubit = context.read<DashboardCubit>();
+    return BlocBuilder<DashboardCubit, DashboardState>(
+      builder: (context, state) {
+        return Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: DashboardTab.values.map((tab) {
+              final isSelected = tab == state.selectedTab;
+              String title;
```

Comment:
a more elegant way:
```
final title = switch(tab) {
 DashboardTab.upcoming => context.localizations.upcoming,
 DashboardTab.score => context.localizations.score,
 DashboardTab.favorites=>context.localizations.favorites,
 }

Reply:
Additionally, creating `title` should be moved to a private method inside this class e.g. `String _getTitle`, you would then call it directly inside the Text widget: `Text(_getTitle())`